eslint.config.mjs

```diff
@@ -419,21 +420,32 @@ export default tseslint.config([
       'turbo/no-undeclared-env-vars': 'off',
     },
   },
-  ...pluginYml.configs['flat/recommended'],
   {
-    name: 'repo/.github',
-    // rules: {
-    // 'regex/invalid': [
-    //   'error',
-    //   [
-    //     {
-    //       regex: '^(?!.*\\$TURBO_ARGS( |$)).*turbo \\S+',
-    //       message: 'Invalid turbo CI command. Must contain `$TURBO_ARGS`',
-    //     },
-    //   ],
-    // ],
-    // },
+    name: 'repo/jsdoc',
+    ...pluginJsDoc.configs['flat/recommended-typescript'],
+    files: ['packages/shared/src/**/*.{ts,tsx}'],
+    ignores: ['**/__tests__/**'],
+    plugins: {
+      jsdoc: pluginJsDoc,
+    },
+    rules: {
+      ...pluginJsDoc.configs['flat/recommended-typescript'].rules,
+      'jsdoc/check-examples': 'off',
+      'jsdoc/informative-docs': 'warn',
+      'jsdoc/check-tag-names': ['warn', { definedTags: ['inline', 'unionReturnHeadings'], typed: false }],
+      'jsdoc/require-hyphen-before-param-description': 'warn',
+      'jsdoc/require-description': 'warn',
+      'jsdoc/require-description-complete-sentence': 'warn',
+      'jsdoc/require-param': ['warn', { ignoreWhenAllParamsMissing: true }],
+      'jsdoc/require-param-description': 'warn',
+      'jsdoc/require-returns': 'off',
```

Comment:
Together with `require-param` I turned these off (unless you define at least one `@param`) since we use TypeScript types in most places. It doesn't hurt to add them but I don't think it's necessary in most places